requirements/runtime.in

```diff
@@ -1,5 +1,11 @@
 twine
 
+# NOTE: This is pulled in transitively through `twine`, but we also declare
+# NOTE: it explicitly here because `oidc-exchange.py` uses it.
+# NOTE: Longer-term, we should use the `id` package directly.
```

Comment:
Will need to regenerate the pins via pip-tools once this happens.

Reply:
Yep! I tried doing this on macOS but got different resolutions (duh), so I'll regenerate this on a Linux machine.

Reply:
I suppose it's not needed right now since the whole tree is there anyway. Best, do this after adding `id`. Also, maybe use the same image and run pip-tools within the container so it generates the pins as closely as possible to the actual runtime.

Reply:
Sounds good! I've added `id`, so I'll rebuild this within the same container in a bit.